src/Sentry/Internal/UnsampledSpan.cs

```diff
@@ -0,0 +1,9 @@
+namespace Sentry.Internal;
+
+internal class UnsampledSpan(UnsampledTransaction transaction, SpanId? spanId = null) : NoOpSpan
```

Comment:
medium: `sealed`

Could we seal this class too? To be in sync with `UnsampledTransaction`.